platform/macos/app/MapDocument.m

```diff
@@ -182,12 +182,6 @@ - (IBAction)showStyle:(id)sender {
         case 6:
             styleURL = [MGLStyle satelliteStreetsStyleURL];
             break;
-        case 7:
-            styleURL = [MGLStyle trafficDayStyleURL];
-            break;
-        case 8:
-            styleURL = [MGLStyle trafficNightStyleURL];
```

Comment:
The menu items themselves are defined in MapDocument.xib (for the toolbar) and MainMenu.xib (for the View menu). Removing the backing code without removing the UI elements leads to the assert below when interacting with the UI elements.